### ISSUE TITLE
Component is being used differently than current experimental design suggestion (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#61757]
#### VFS team name
Appointments

#### VFS product name
Appointment List

#### VFS feature name
Print List

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
Tertiary buttons were designed originally to work with cards and not on their own.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Work with Matt Dingee on whether or not this can be used on its own. Look at the styling of the [tertiary button](https://www.sketch.com/s/be292806-8dbf-4201-bfa7-0edbba103347/v/Wraowk) in the card (all caps and hover states). Look at the example on the [card](https://design.va.gov/components/card#financial-status-report) and [close](https://design.va.gov/components/notification#with-close-icon-button) on the modal.

In particular, note that the all-caps styling for tertiary buttons was a request from accessibility to help establish a pattern that distinguishes tertiary buttons from links.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by outerpress** [comment]
Notes from 07/19 collab cycle office hours:
* We'll release the button as designed and start gathering click analytics.
* @humancompanion-usds will work on iterations to the button/link situation - I'll send a Sketch version of our detail page redesign WIP so he can explore the more complex use-cases
* Once an iteration is ready, we'll modify the print button and compare analytics.
* When we're ready with all MHV on VA.gov features slated for September, we'll run UAT which will include the print button.
* We'll report back to design council with our findings for that, and decide if further iteration is needed.

Let me know if I missed anything @allison0034 @humancompanion-usds

**Comment by outerpress** [comment]
We matched the styling in #61769 so this one should be good to go @ldelacosta

**Comment by ldelacosta** [comment]
Completed the work. Closing out the ticket. cc: @JRRoof
